modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -57,21 +56,9 @@ apiVersion: "k8s.cni.cncf.io/v1"
 ----
 <1> The type is `bond`.
 <2> The `ifname` attribute specifies the name of the bond interface.
-<3> The `mode` attribute specifies the bonding mode. 
-+
-[NOTE]
-====
-The bonding modes supported are:
-
-* `balance-rr` - 0 
-* `active-backup` - 1
-* `balance-xor` - 2
-
-For `balance-rr` or `balance-xor` modes, you must set the `trust` mode to `on` for the SR-IOV virtual function.
-====
-<4> The `failover` attribute is mandatory for active-backup mode.
-<5> The `linksInContainer=true` flag informs the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV and Multus.
-<6> The `links` section defines which interfaces will be used to create the bond. By default, Multus names the attached interfaces as: "net", plus a consecutive number, starting with one.
+<3> The `failover` attribute is mandatory for active-backup mode.
+<4> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV/Multus.
```

Comment:
```suggestion
<4> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default, Bond CNI looks for these interfaces on the host, which does not work for an integration with SRIOV/Multus.
```
I stumbled a bit on this sentence. Does my suggestion change the technical accuracy of the sentence?